src/ReactLeafletMap/layers/WmsLayer/SingleTileLayer.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types';
+import {createLayerComponent} from '@react-leaflet/core';
+import wms from './leaflet.wms';
+
+function createLeafletElement(props, context) {
+	const {pane, url, crs, params} = props;
```

Comment:
Where does pane prop come from?

Reply:
It was part of magic who I found. Pane is injected by ReactLeaflet.